js-sandbox/src/script.rs

```diff
@@ -8,10 +8,11 @@ use std::{thread, time::Duration};
 use deno_core::{op, Extension, JsRuntime, OpState, ZeroCopyBuf};
 use serde::de::DeserializeOwned;
 
-use crate::call_args::CallArgs;
-use crate::{AnyError, JsValue};
+use crate::{AnyError, CallArgs, JsError, JsValue};
 
+/// Trait denoting a JS API
```

Comment:
Added missing trait doc, but you may want to revise it

Reply:
It's just a placeholder to get rid of the warning so the ci passes

Honestly not certain what that trait does

Reply:
What do you mean? We don't enforce symbols to be documented in CI. 

This trait hasn't been documented for the longest time...